src/main/java/tetris/KeyBindings.java

```diff
@@ -8,7 +8,7 @@
 import javafx.scene.input.KeyCode;
 
 public final class KeyBindings implements Iterable<Entry<KeyCode, String>> {
-    private HashMap<KeyCode, String> bindings = new HashMap<KeyCode, String>();
+    private HashMap<KeyCode, String> bindings = new HashMap<>();
```

Comment:
Did some research: Apparently Java is also moving more and more to a loosely typed language (thank god). In Java 7+ these implicit typecasts are resolved at compile-time, so there actually doesn't happen any casting. This resolution is called _type inference_. Source: https://docs.oracle.com/javase/tutorial/java/generics/genTypeInference.html